packages/vscale/src/ordinal-scale.ts

```diff
@@ -24,6 +42,10 @@ export class OrdinalScale extends BaseScale implements IBaseScale {
   }
 
   scale(d: any): any {
+    const special = this._getSpecifiedValue(d);
+    if (special) {
```

Comment:
how about zero or blank space is a specified value ?

Reply:
o ~yes , i will fix this